packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -444,9 +443,9 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
   ) {
     channelService.streamCommandOutput(execution);
     channelService.showChannelOutput();
-    notificationService.reportExecutionError(
```

Comment:
can you explain this change please :)

Reply:
so the function we were calling before was 'reportExecutionError', which as the name says is used for reporting errors. I just changed it to the other function that is used to report general command execution

Reply:
got it. so now we will get a notification on command execution status?